AISKU/scripts/publishReleaseToCdn.ps1

```diff
@@ -263,18 +263,19 @@ Function GetReleaseFiles
 Function GetVersion(
     [string] $name
 ) {
-    $regMatch = '^(.*\/)*([^\/\d]*\.)(\d+(\.\d+)*)(\.(?:js|min\.js)(?:\.map)?)$'
+    $regMatch = '^(.*\/)*([^\/\d]*\.)(\d+(\.\d+)*(-[^\.]+)?)(\.(?:js|min\.js)(?:\.map)?)$'
```

Comment:
Support publishing -beta; -alpha; etc releases to the major/minor versions (v2; v2.x)